covid_api/api/utils.py

```diff
@@ -223,9 +243,11 @@ def get_zonal_stat(geojson: Feature, raster: str) -> Tuple[float, float]:
         # calculate the coverage of pixels for weighting
         pctcover = rasterize_pctcover(geom, atrans=window_affine, shape=data.shape[1:])
 
+        masked_data = np.ma.masked_values(data[0], src.nodatavals[0])
```

Comment:
@leothomas is it intentional that you're not using rasterio native numpy masked array support? 

e.g 
```python
data = src.read(window=window, masked=True)
```

Reply:
Oh wow - no, not at all intentional. I immediately jumped in to masking out `nodata` values in the calculation, as opposed to `rasterio` . 

I'll give this a try, makes for a much easier fix! 

Thanks!

Reply:
@vincentsarago Since the `maked` flag changes the type of the returned data from `<class 'numpy.ndarray'>` to `<class 'numpy.ma.core.MaskedArray'>`, is it correct to still use `np.ma.average` and `np.ma.median` (instead of `np.average` and `np.nanmedian`)?

Reply:
@leothomas I don't know but I think it's safer and clearer to use `numpy.ma` functions